src/integration/config/compile.yml

```diff
@@ -42,6 +42,7 @@ global:
         - +define+CPTRA_TOP_PATH=caliptra_ss_top.caliptra_top_dut
         - +define+MCU_RV_BUILD_AXI4
         - +define+MCU_RV_OPENSOURCE
+        - +define+MCU_RV_DCCM_ENABLE
```

Comment:
Isn't this the wrong name? Thought it should be `css_mcu0_RV_DCCM_ENABLE `. It should be taken care of in the veer common_defines already.

Reply:
Removed the line. :)